google-cloud-firestore/lib/google/cloud/firestore/query.rb

```diff
@@ -940,6 +941,26 @@ def get
         end
         alias run get
 
+        ##
+        # Creates an AggregateQuery object for the query.
+        #
+        # @return [AggregateQuery] New empty aggregate query.
+        #
+        # @example
+        #   require "google/cloud/firestore"
+        #
+        #   firestore = Google::Cloud::Firestore.new
+        #
+        #   # Get a collection reference
+        #   query = firestore.col "cities"
+        #
+        #   # Create an aggregate query
+        #   aggregate_query = query.aggregate_query
+        #
+        def aggregate_query
+          AggregateQuery.new query, [], parent_path, client
```

Comment:
Good point. I added a factory method in `AggregateQuery` class to be used by the `Query` class. This will avoid exposing the internal property `aggregates`.